akka-stream/src/main/java-jdk9/akka/stream/javadsl/Jdk9Sink.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com/>
+ */
+
+package akka.stream.javadsl;
+
+/**
+ * TODO java doc
+ */
+public interface Jdk9Sink {
```

Comment:
Would the name `Jdk9Sink` get confusing once JDK 10 is out?

Reply:
Yeah, names are pretty bad still... I'm torn between `JavaFlowSinkSupport` or something similar...